site/data/docs/latest-toc.yml

```diff
@@ -0,0 +1,93 @@
+toc:
```

Comment:
The menu is manual, so we would need to add new featured documentation items by hand.